apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/sharing-modal/index.tsx

```diff
@@ -72,7 +71,6 @@ const SharingModal: React.FC = () => {
 			! shouldShowFirstPostPublishedModal &&
 			! shouldShowSellerCelebrationModal &&
 			! shouldShowVideoCelebrationModal &&
-			launchpadScreenOption !== 'full' &&
```

Comment:
This reads that this modal will start showing up in cases where we might currently expect the launchpad modal? Im wondering if this is unintended behavior for that phase of onboarding? cc @daledupreez

Reply:
The problem is that launchpadScreenOption doesn't seem to change from `full` after we've published the first post. hmmm I'll take a look, there must be some other flag we can use here.

Reply:
> doesn't seem to change from full after we've published the first post.

It wouldn't change after they publish the first post, but it would change after they complete or permanently dismiss the fullscreen launchpad (may have a few other tasks than publishing a post). So I think the LP modal would show up until that happens?  I think theres definitely some room to rethink consolidate these many modals though... 😕